pkg/i2gw/providers/istio/converter.go

```diff
@@ -673,14 +693,14 @@ func (c *converter) convertVsHTTPRoutes(virtualService metav1.ObjectMeta, istioH
 			httpRoutesWithRewrites := c.createHTTPRoutesWithRewrite(createHTTPRouteParams, httpRoute.GetRewrite(), httpRouteFieldPath.Child("HTTPRewrite"))
 			resHTTPRoutes = append(resHTTPRoutes, httpRoutesWithRewrites...)
 			for _, httpRoute := range httpRoutesWithRewrites {
-				notify(notifications.InfoNotification, fmt.Sprintf("Virtual Service \"%v\"/\"%v\" successfully converted to HTTPRoute \"%v\"/\"%v\"", vs.Namespace, vs.Name, httpRoute.Namespace, httpRoute.Name))
+				notify(notifications.InfoNotification, fmt.Sprintf("successfully converted to HTTPRoute \"%v/%v\"", httpRoute.Namespace, httpRoute.Name), vs)
 			}
 			continue
 		}
 
 		httpRoute := c.createHTTPRoute(createHTTPRouteParams)
 		resHTTPRoutes = append(resHTTPRoutes, httpRoute)
-		notify(notifications.InfoNotification, fmt.Sprintf("Virtual Service \"%v/%v\" successfully converted to HTTPRoute \"%v\"/\"%v\"", vs.Namespace, vs.Name, httpRoute.Namespace, httpRoute.Name))
+		notify(notifications.InfoNotification, fmt.Sprintf("successfully converted to HTTPRoute \"%v/%v\"", httpRoute.Namespace, httpRoute.Name), vs)
```

Comment:
these changes are intentional?

Reply:
Yes this one is intentional, for unirformity for all success messages